### PR TITLE
feat(pci.db): add equivalent api on fork page

### PR DESCRIPTION
**Pull request by abullet33**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master` 
| Bug fix?         | yes
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #datatr-190 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Description
Adding API equivalent in fork page and modifying API calls to deal deprecated change on API.

### 🎏TRANSLATION
- [x] TRANS-52567

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/storages/databases/add/add.utils.js [hunk 1]
```diff
@@ -14,6 +16,21 @@ export function getOrderDataFromModel(model) {
...
+        serviceId: model.databaseId,
+        backupId: model.backupId,
+      };
+    } else {
+      orderData.forkFrom = {
+        serviceId: model.databaseId,
+        pointInTime: model.timestamp,
+      };
+    }
+  }
```
Comment (ghyenne) [inline packages/manager/modules/pci/src/projects/project/storages/databases/add/add.utils.js:32]:
```suggestion
  if (model.restoreMode) {
      orderData.forkFrom = {
        serviceId: model.databaseId,
        ...(model.restoreMode === RESTORE_MODES.BACKUP ? {backupId: model.backupId} : {pointInTime: model.timestamp})
      };
  }
```

File: packages/manager/modules/pci/src/projects/project/storages/databases/database/backups/fork/command/command.controller.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+import { ORDER_KEYS } from '../fork.constants';
+
+export default class ForkCommandCtrl {
+  /* @ngInject  */
+  constructor($translate, DatabaseService) {
+    this.$translate = $translate;
+    this.orderKeys = ORDER_KEYS;
+    this.DatabaseService = DatabaseService;
+  }
+
```
Comment (ghyenne) [inline packages/manager/modules/pci/src/projects/project/storages/databases/database/backups/fork/command/command.controller.js:16]:
```suggestion
  constructor() {
    this.orderKeys = ORDER_KEYS;
  }

```
I think doesn't used.

File: packages/manager/modules/pci/src/projects/project/storages/databases/database/backups/fork/command/translations/Messages_fr_FR.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{
+  "pci_databases_fork_command_close_button": "Fermer",
+  "pci_databases_fork_command_title": "Commande équivalente",
+  "pci_databases_fork_command_description": "La commande suivante vous permet de créer votre service grâce à notre API.",
+  "pci_databases_fork_command_description_terraform": "La resource suivante vous permet de créer votre service grâce à notre Terraform OVH Provider.",
```
Comment (ghyenne) [inline packages/manager/modules/pci/src/projects/project/storages/databases/database/backups/fork/command/translations/Messages_fr_FR.json:5]:
```suggestion
  "pci_databases_fork_command_description_terraform": "La resource suivante vous permet de créer votre service grâce à notre Terraform OVHcloud Provider.",
```
I don't know if it's marketing name or if you can change OVH by OVHcloud

File: packages/manager/modules/pci/src/projects/project/storages/databases/add/add.utils.js (hunk 1 above)
Comment (abullet33) [reply packages/manager/modules/pci/src/projects/project/storages/databases/add/add.utils.js]: Hi, I forget a case here, so I went for a switch instead. Sorry.

File: packages/manager/modules/pci/src/projects/project/storages/databases/database/backups/fork/command/command.controller.js (hunk 2 above)
Comment (abullet33) [reply packages/manager/modules/pci/src/projects/project/storages/databases/database/backups/fork/command/command.controller.js]: Hello, totally right, fix :)

File: packages/manager/modules/pci/src/projects/project/storages/databases/database/backups/fork/fork.controller.js [hunk 4]
```diff
@@ -361,6 +361,15 @@ export default class {
     );
   }
 
+  goToAPICommand() {
+    this.model.databaseId = this.database.id;
+    this.model.disk = {
+      initialSize: this.model.flavor.minDiskSize,
+      additionalDiskSize: this.model.additionalDiskSize,
+    };
+    this.goToCommand(this.model);
```
Comment (ghyenne) [inline packages/manager/modules/pci/src/projects/project/storages/databases/database/backups/fork/fork.controller.js:370]:
```suggestion
    return this.goToCommand(this.model);
```

**Review by ghyenne — DISMISSED** [review]

File: packages/manager/modules/pci/src/projects/project/storages/databases/database/backups/fork/fork.controller.js (hunk 4 above)
Comment (abullet33) [reply packages/manager/modules/pci/src/projects/project/storages/databases/database/backups/fork/fork.controller.js]: You are right! Done

**Review by jperchoc — APPROVED** [review]

**Review by ghyenne — DISMISSED** [review]

**Review by jperchoc — DISMISSED** [review]

**Review by ghyenne — APPROVED** [review]

**Review by anooparveti — APPROVED** [review]

**Review by jperchoc — APPROVED** [review]

**Review by anooparveti — APPROVED** [review]

File: packages/manager/modules/pci/src/projects/project/storages/databases/database/backups/fork/command/command.html [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+    <p data-translate="pci_databases_fork_command_parameters"></p>
+    <pre data-ng-bind="$ctrl.apiData | json" class="monospace"></pre>
+    <p data-translate="pci_databases_fork_command_parameters_info"></p>
+    <dl>
+        <div
+            data-ng-repeat="orderKey in $ctrl.orderKeys track by $index"
+            class="row mb-2"
+        >
+            <dt
+                class="col-6 col-md-4 ont-weight-normal"
```
[the remark this answers is not in the document]
Comment (abullet33) [reply packages/manager/modules/pci/src/projects/project/storages/databases/database/backups/fork/command/command.html]: Fix In PR

**Pull request merged**